docker/Dockerfile

```
@@ -104,11 +107,12 @@ COPY --from=us-docker.pkg.dev/zealous-zebra/zebra/lightwalletd /opt/lightwalletd
 # This is the caching Docker layer for Rust!
 #
 # TODO: is it faster to use --tests here?
-RUN cargo chef cook --release --features "sentry ${TEST_FEATURES} ${FEATURES}" --workspace --recipe-path recipe.json
+RUN cargo chef cook --release --features "${TEST_FEATURES} ${FEATURES}" --workspace --recipe-path recipe.json
```

Could we add 'sentry' in to this features argument in release-binaries.yml:
https://github.com/ZcashFoundation/zebra/blob/8a9113c4e2fc3cd551f5cf94c3b8177591a54eae/.github/workflows/release-binaries.yml#L49

Nitpick:
```suggestion
RUN cargo chef cook --release --features "${ENTRYPOINT_FEATURES}" --workspace --recipe-path recipe.json
```

---

> Could we add 'sentry' in to this features argument in release-binaries.yml:

Done!

> Nitpick:

It's meant to be separate so it can be overridden separately. I'll add a note.